posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -37,7 +37,9 @@ def to_query(self) -> ast.SelectQuery:
 
     def select(self):
         exprs: list[ast.Expr] = [
-            ast.Alias(alias="occurrences", expr=ast.Call(name="count", args=[])),
+            ast.Alias(
+                alias="occurrences", expr=ast.Call(name="count", distinct=True, args=[ast.Field(chain=["uuid"])])
```

Comment:
@daibhin found that due to the `left outer join` for overrides, sometimes events are double counted, which makes this new test I added flakey. Not sure why this doesn't happen on existing tests 🤔 , maybe to do with how/when overrides are set, since I add events after the first flush.

But anyhow, should make this case clear where we never doublecount events.

Reply:
Nice! Nothing springs to mind as to why the over counting is flakey but looks like the distinct check should prevent it for now